libtcmu.c

```diff
@@ -765,6 +765,36 @@ uint32_t tcmu_get_dev_max_xfer_len(struct tcmu_device *dev)
 	return dev->max_xfer_len;
 }
 
+/**
+ * tcmu_set/get_dev_opt_unmap_gran - set/get device's optimal unmap granularity
```

Comment:
Actually ignore that comment. It probably sounds just as weird using the grain naming.